dns2tcp.c

```diff
@@ -389,6 +426,12 @@ static void udp_recvmsg_cb(evloop_t *evloop, evio_t *watcher __unused, int event
     if (sockfd < 0)
         goto free_ctx;
 
+    if (has_opt(OPT_BIND_TCP)) {
+        if (bind(sockfd, &g_bind_skaddr.sa, skaddr_len(&g_bind_skaddr)) < 0) {
+            log_error("bind tcp address: %m");
+            goto close_sockfd;
+        }
+    }
```

Comment:
Add a blank line above `if (connect(...)) {` to separate the two statement blocks.